server/index.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const morgan = require('morgan');
+const db = require('../database/index');
+
+
+const app = express();
+app.use(bodyParser.json());
+app.use(morgan('dev'));
+
+let staticPath = __dirname + '/../public';
+console.log(staticPath);
+app.use('/books/:id', express.static(staticPath));
+
+// app.get('/books/:id', (req, res) => {
+//   const id = req.params.id;
+//   console.log(id);
+//   console.log('STATIC PAGE');
+//   res.send('hello get book id');
+// });
+
+//get initinal details
+app.get('/books/:id/details', (req, res) => {
+  const id = req.params.id;
+  db.getDetails(id)
+    .then(results => {
+      let details = results[0][0];
```

Comment:
you may need to verify that results have a length greater than "0" before going ahead to index it in order to prevent the "undefined is not an object error"

Reply:
Thank you! I will fix this right away!